api/nakadi-event-bus-api.yaml

```diff
@@ -2328,6 +2386,61 @@ definitions:
         description: |
           Number of milliseconds that Nakadi stores events published to this event type.
 
+  EventTypeAuthorization:
+    type: object
+    description: |
+      Authorization section for an event type. This section defines three access control lists: one for producing
+      events ('write'), one for consuming events ('read'), and one for administering an event type ('admin').
+      Regardless of the values of the authorization properties, administrator accounts will always be authorized.
+    properties:
+      admin:
+        type: array
+        description: |
+          An array of subject attributes that are required for updating the event type. Any one of the attributes
+          defined in this array is sufficient to be authorized. The catch-all item takes precedence over all others,
+          i.e. if it is present, all users are authorized.
+        items:
+          $ref: '#/definitions/AccessControlListItem'
+        minItems: 1
+      read:
+        type: array
+        description: |
+          An array of subject attributes that are required for reading events from the event type. Any one of the
+          attributes defined in this array is sufficient to be authorized. The catch-all item takes precedence over
+          all others, i.e., if it is present, all users are authorized.
+        items:
+          $ref: '#/definitions/AccessControlListItem'
+        minItems: 1
+      write:
+        type: array
+        description: |
+          An array of subject attributes that are required for writing events to the event type. Any one of the
+          attributes defined in this array is sufficient to be authorized. The catch-all item takes precedence over
+          all others, i.e. if it is present, all users are authorized.
+        items:
+          $ref: '#/definitions/AccessControlListItem'
+        minItems: 1
+    required:
+      - admin
+      - read
+      - write
+
+  AccessControlListItem:
```

Comment:
We did call them roles, but... they're not roles :)
So I want to use another name. I think `AuthorizationAttribute` is better?